alphadia/workflow/base.py

```diff
@@ -141,20 +145,10 @@ def load(
 
         self.reporter.log_event("section_stop", {})
 
-    @property
-    def instance_name(self) -> str:
-        """Name for the particular workflow instance. this will usually be the name of the raw file"""
-        return self._instance_name
-
-    @property
-    def quant_path(self) -> str:
```

Comment:
what is the reason that these two were removed but .path not? just to slim down the interface? Would be fine by me, just curious.